### ISSUE TITLE
cupertino_http: `BaseResponse.request` is null

### ISSUE THREAD

**Issue by hoc081098** [dart-lang/http#782]
```dart
  final http.StreamedResponse res =
      await CupertinoClient.defaultSessionConfiguration().send(
    http.Request(
        'GET', Uri.parse('https://jsonplaceholder.typicode.com/users/1')),
  );
  print('res.request: ${res.request}');
```

Console:
```
flutter: res.request: null
```

**Comment by brianquinlan** [comment]
This is fixed in the cupertino_http 0.0.5 release.

**Comment by hoc081098** [comment]
@brianquinlan thanks for that

_Sent from my 2201117TG using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
